src/app/components/Feedback/Feedback.jsx

```diff
@@ -84,16 +126,24 @@ class Feedback extends React.Component {
                 </label>
                 <textarea
                   id="feedback-textarea-comment"
-                  name="entry.148983317"
-                  rows="5"
+                  name="Feedback"
+                  value={fields.Feedback}
                   ref={this.commentText}
+                  rows="5"
                   aria-required="true"
                   tabIndex="0"
+                  onChange={e => this.handleInputChange(e)}
```

Comment:
I think you can just do `onChange={this.handleInputChange}`, right? A subtle change but passing in an anonymous function creates a new function on every render.